core/client/fs/src/main/java/alluxio/client/file/RetryHandlingFileSystemMasterClient.java

```diff
@@ -312,4 +310,20 @@ public void updateUfsMode(final AlluxioURI ufsUri,
             .setUfsPath(ufsUri.getRootPath()).setOptions(options).build()),
         "UpdateUfsMode");
   }
+
+  /**
+   * Gets the path that will be transported to master.
+   * Scheme-less URIs are assumed to be Alluxio paths,
+   * and getPath() is used to avoid string conversion.
+   *
+   * @param uri uri
+   * @return transport path
+   */
+  private String getTransportPath(AlluxioURI uri) {
```

Comment:
It looks useful. How about make this a member method of `AlluxioURI` rather than a util method here (and if it stays here, I would even make it static method).

Reply:
I couldn't find a generic use for this in `AlluxioURI`. Made it static instead.